book/src/wildcard-audit-entries.md

```diff
@@ -27,7 +28,11 @@ that releases are always cut from a branch for which every change has been
 approved by a trusted individual who will enforce the criteria.
 
 Wildcard audits can be added with `cargo vet certify` using the `--wildcard`
-option.
+option. This sets the `end` date to one year in the future. Once added (whether
+manually or by `cargo vet certify --wildcard`), the `end` date can be updated to
+one year in the future using the `cargo vet renew CRATE` command. `cargo vet
+renew --expiring` can be used to automatically update _all_ audits which would
+expire in the next six weeks or have already expired.
```

Comment:
I'll reword that sentence. Currently there is no flag to change the end date for `renew` though, it's always 1 year. That could be changed.

I agree, I think it may make sense to set `suggest-renewal = false` if an explicit end date is provided.